.travis.yml

```diff
@@ -4,9 +4,9 @@ dist: focal
 git:
   submodules: true
 before_install:
-  - repo_checks/submodule_check.sh
+  # - repo_checks/submodule_check.sh
+  - "true"
 install:
-  - git submodule update --remote
```

Comment:
Can we please just not have this? It leads to a situation where the CI is testing something different than the repository is stating to contain. Submodules are good to encode a known to work combination of commits in different repositories and to represent a certain state of source checkout across multiple projects. 
If this were the desired behaviour, we should just stop using submodules here. Maybe, by the time being, it is, as we can pull the installer collection from Galaxy now.

Ratinale: Submodules are good to encode a known to work combination of commits in different repositories and to represent a certain state of source checkout across multiple projects. This command breaks that bond.